### ISSUE TITLE
Install script --optional-features and --oci-cli-version parameters incompatible

### ISSUE THREAD

**Issue by alexdyas** [oracle/oci-cli#370]
Hello,

Combining the `--optional-features` and `--oci-cli-version` thus for a promptless install :

```
./install.sh --install-dir /home/foo/usr/local/lib --exec-dir /home/foo/usr/local/bin --script-dir /home/foo/usr/local/bin --oci-cli-version 2.21.0 --optional-features none --no-tty
```

results in an error:

```
...
-- Executing: ['/home/foo/usr/local/lib/bin/pip', 'install', '--cache-dir', '/tmp/tmprdirnmmg', 'oci_cli==2.21.0[none]', '--upgrade']
ERROR: Extras after version '==2.21.0[none]'.
Traceback (most recent call last):
  File "/tmp/oci_cli_install_tmp_0MEX", line 685, in <module>
    main()
  File "/tmp/oci_cli_install_tmp_0MEX", line 640, in main
    install_cli(install_dir, tmp_dir, args.oci_cli_version, OPTIONAL_FEATURES)
  File "/tmp/oci_cli_install_tmp_0MEX", line 298, in install_cli
    exec_command(cmd, env=env)
  File "/tmp/oci_cli_install_tmp_0MEX", line 141, in exec_command
    subprocess.check_call(command_list, cwd=cwd, env=env)
  File "/usr/lib64/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/home/foo/usr/local/lib/bin/pip', 'install', '--cache-dir', '/tmp/tmprdirnmmg', 'oci_cli==2.21.0[none]', '--upgrade']' returned non-zero exit status 1.
```
Somehow the optional features argument is being tacked on to the version option.

Workaround (for no optional features) :
```
echo | ./install.sh --install-dir /home/foo/usr/local/lib --exec-dir /home/foo/usr/local/bin --script-dir /home/foo/usr/local/bin --oci-cli-version 2.21.0 --no-tty
```

Thanks,

Alexander

**Comment by vish1** [comment]
@alexdyas Sorry for the delay. Thanks for reporting. We will look into this issue and provide an update.

**Comment by kernleee** [comment]
Fixed in the latest release, 3.3.2. Closing this issue.

**Comment by alexdyas** [comment]
Great, thanks guys.
